src/ValueObject/Route/AuthToken.php

```diff
@@ -27,6 +27,6 @@ public function __construct()
     {
         $this->paramNameList = array();
         $this->identifier    = '/auth/token';
-        $this->pattern       = '/^\/auth\/token\/*$/';
+        $this->pattern       = '/\/auth\/token\/*$/';
```

Comment:
Although not very likely here, this method of pattern matching can cause misdetection. We should make the patterns work the same (see /v2/) - defining a beginning and an end ot the pattern.